beacon_node/beacon_chain/src/schema_change/migration_schema_v26.rs

```diff
@@ -0,0 +1,83 @@
+use crate::persisted_custody::{PersistedCustody, CUSTODY_DB_KEY};
+use crate::validator_custody::CustodyContextSsz;
+use crate::BeaconChainTypes;
+use ssz::{Decode, Encode};
+use ssz_derive::{Decode, Encode};
+use std::sync::Arc;
+use store::{DBColumn, Error, HotColdDB, KeyValueStoreOp, StoreItem};
+use tracing::info;
+
+#[derive(Debug, Encode, Decode, Clone)]
+pub(crate) struct CustodyContextSszV1 {
+    pub(crate) validator_custody_at_head: u64,
+    pub(crate) persisted_is_supernode: bool,
+}
+
+pub(crate) struct PersistedCustodyV1(CustodyContextSszV1);
+
+impl StoreItem for PersistedCustodyV1 {
+    fn db_column() -> DBColumn {
+        DBColumn::CustodyContext
+    }
+
+    fn as_store_bytes(&self) -> Vec<u8> {
+        self.0.as_ssz_bytes()
+    }
+
+    fn from_store_bytes(bytes: &[u8]) -> Result<Self, Error> {
+        let custody_context = CustodyContextSszV1::from_ssz_bytes(bytes)?;
+        Ok(PersistedCustodyV1(custody_context))
+    }
+}
+
+/// Upgrade the `CustodyContext` entry to v26.
+pub fn upgrade_to_v26<T: BeaconChainTypes>(
+    db: Arc<HotColdDB<T::EthSpec, T::HotStore, T::ColdStore>>,
+) -> Result<Vec<KeyValueStoreOp>, Error> {
+    let res = db.get_item::<PersistedCustodyV1>(&CUSTODY_DB_KEY);
+    let ops = match res {
```

Comment:
Do you want to try deleting it if Fusaka is not scheduled? That might give us a bit more freedom to experiment with the format, which committing to writing data on non-Fusaka nodes

Reply:
OTOH, it's simpler if the DB schema isn't fork-dependent

Reply:
yeah good point, i'll add that

Reply:
Updated